src/app/modules/shared/components/header/header.component.html

```diff
@@ -1,12 +1,12 @@
-<div>
+<div *ngIf="!authService.hideHeader">
```

Comment:
can store it in local variable of component, as we already have other used other things of library like that

Reply:
Taken it as a service variable reason is when we hit test link in new tab then header component loads first and then the general instructions component loads. So, if we use local variable then we can not hide the header. That's the reason taken service variable